gradio/components.py

```diff
@@ -2535,7 +2535,7 @@ def __init__(
             headers: List of str header names. If None, no headers are shown.
             row_count: Limit number of rows for input and decide whether user can create new rows. The first element of the tuple is an `int`, the row count; the second should be 'fixed' or 'dynamic', the new row behaviour. If an `int` is passed the rows default to 'dynamic'
             col_count: Limit number of columns for input and decide whether user can create new columns. The first element of the tuple is an `int`, the number of columns; the second should be 'fixed' or 'dynamic', the new column behaviour. If an `int` is passed the columns default to 'dynamic'
-            datatype: Datatype of values in sheet. Can be provided per column as a list of strings, or for the entire sheet as a single string. Valid datatypes are "str", "number", "bool", and "date".
+            datatype: Datatype of values in sheet. Can be provided per column as a list of strings, or for the entire sheet as a single string. Valid datatypes are "str", "number", "bool", "date", and "markdown".
```

Comment:
I feel like we should only accept "markdown" as a value since that's the only team we really change the user data?

Reply:
I think it's useful to have different data types for the Dataframe component when it is used as an input component. For example, we could preprocess the data in different ways depending on what the column type is (I actually thought we did that, but I don't see it in the code), or we might do same basic validation in the future